configure

```diff
@@ -2851,12 +2851,18 @@ fi
 
 
 #!/bin/sh
-if test -d ./src/boost.tar.gz;
+if test "$(uname -m)" == "x86_64";
   then
-      echo 'found boost header sources and tar archive;\n using what is there.'
-  else
-      echo "untarring boost include tree...";
-      tar -zxf ./src/boost.tar.gz -C ./src;
+        CPPFLAGS="-I. ${CPPFLAGS}"
```

Comment:
Add the current folder to the includes path only when it has to use the local copy of boost.
This change is optional but IMO it makes it better. Local header files should be included with `"..."` instead of `<...>`.
Please let me know if you have any concerns and I will remove it!